tonic/Cargo.toml

```diff
@@ -26,24 +26,28 @@ version = "0.11.0"
 codegen = ["dep:async-trait"]
 gzip = ["dep:flate2"]
 zstd = ["dep:zstd"]
-default = ["transport", "codegen", "prost"]
+default = ["channel", "codegen", "prost"]
 prost = ["dep:prost"]
 tls = ["dep:rustls-pemfile", "transport", "dep:tokio-rustls", "dep:tokio", "tokio?/rt", "tokio?/macros"]
 tls-roots = ["tls-roots-common", "dep:rustls-native-certs"]
-tls-roots-common = ["tls"]
+tls-roots-common = ["tls", "channel"]
 tls-webpki-roots = ["tls-roots-common", "dep:webpki-roots"]
 router = ["dep:axum"]
 transport = [
   "router",
   "dep:async-stream",
-  "channel",
   "dep:h2",
-  "dep:hyper", "dep:hyper-util", "dep:hyper-timeout",
+  "dep:hyper", "dep:hyper-util",
   "dep:socket2",
   "dep:tokio", "tokio?/macros", "tokio?/net", "tokio?/time",
-  "dep:tower",
+  "dep:tower", "tower?/util", "tower?/limit",
+]
+channel = [
```

Comment:
Can you update the crate root documentation to clarify a bit what the distinction is between `channel` and `transport`? I don't think the current docs are very clear:

> `transport`: Enables the fully featured, batteries included client and server implementation based on [hyper](https://docs.rs/hyper), [tower](https://docs.rs/tower) and [tokio](https://docs.rs/tokio). Enabled by default.
> `channel`: Enables just the full featured channel/client portion of the transport feature.